rtv.go

```diff
@@ -0,0 +1,85 @@
+package amppackager
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/robfig/cron"
+)
+
+// die is an alias for log.Fatalf, which can be overridden for testing purposes.
+var die = log.Fatalf
+
+// not a const solely for testing purposes (be able to override this value).
+var rtvHost = "https://cdn.ampproject.org"
+
+// RTVCache stores the AMP runtime version number and the CSS for that version
+type RTVCacheStruct struct {
+	RTV, CSS string
+}
+
+var c = cron.New()
+var RTVCache = new(RTVCacheStruct)
+
+// StartCron starts an hourly cron job to periodically re-fill the RTVCache.
+func StartCron() error {
+	if err := c.AddFunc("@every 1h", rtvPoll); err != nil {
+		return err
+	}
+	c.Start()
```

Comment:
Based on the documentation I'm pretty sure it will run as soon as Start() is called. But I plan to test that when I wire this into main.go in a future CL.